fast/stages/02-networking-vpn/data/subnets/prod/prod-dp-trf-ew1.yaml

```diff
@@ -0,0 +1,5 @@
+# skip boilerplate check
+
+region: europe-west1
+description: Default subnet for dev Data Platform - Transformation layer Dataflow
+ip_cidr_range: 10.20.1.0/24
```

Comment:
dev-dp-orc-ew1.yaml uses a CIDR in the `172.16/12` range, is this supposed to be different?